### ISSUE TITLE
Use Chrome instead of Phantom.js

### ISSUE THREAD

**Issue by andreicek** [webpack-contrib/webpack-canary#29]

**Comment by TheLarkInn** [comment]
Also if this hasn't started chrome has released their own remote desktop protocol abstraction called "puppeteer" something that might also be a choice. (If not already being worked on)

**Comment by andreicek** [comment]
Yup! I've started some work on this but soon switched to Puppeteer since it looks easier to use. Thanks, Sean!
